code/modules/mob/living/carbon/human/human_say.dm

```diff
@@ -1,5 +1,21 @@
+/mob/living/carbon/human/say(message, bubble_type, list/spans, sanitize, datum/language/language, ignore_spam, forced, filterproof, message_range, datum/saymode/saymode)
```

Comment:
please please please please doc this and all the parameters 
Yes I know it;s copied but touch is touch

Reply:
https://discord.com/channels/326822144233439242/326831214667235328/1048033813294489722

Reply:
oh rigth I didnt notice